Source/Lib/Common/Codec/EbThreads.c

```diff
@@ -31,12 +31,16 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <pthread.h>
-#include <semaphore.h>
 #include <unistd.h>
-#endif // _WIN32
 #ifdef __APPLE__
 #include <dispatch/dispatch.h>
-#endif
+typedef void sem_t;
+static INLINE int sem_wait(sem_t* psem) { return (int)dispatch_semaphore_wait((dispatch_semaphore_t)psem, DISPATCH_TIME_FOREVER); }
```

Comment:
```suggestion
static inline int sem_wait(sem_t* psem) { return (int)dispatch_semaphore_wait((dispatch_semaphore_t)psem, DISPATCH_TIME_FOREVER); }
```

since we are using c99, `inline` is properly handled, so no need to use the defined `INLINE` for the inline extension

Reply:
`inline` should be fine here, it is not in cross-platform code though as MSVC does not or only recently learned about inline. That said, in general it would be better to consistently use `inline` everywhere and just `#define` that properly when missing for some MSVC versions.